README.md

```diff
@@ -194,7 +179,11 @@ genres = [
 
 # Initialization of variables
 df = pd.DataFrame(columns=['Title', 'Popularity %'])
-selected_genre = None
+selected_genre = 'Action'
```

Comment:
Use double-quotes everywhere possible. In the entire code.
'Action' -> "Action"
(same thing twice in the line before)